.github/workflows/cpp-ci.yml

```diff
@@ -12,18 +12,16 @@ jobs:
     runs-on: ubuntu-latest
 
     env:
-      CXX: clang++-9
-      CLANG_FORMAT: clang-format-9
+      CXX: clang++-12
+      CLANG_FORMAT: clang-format-12
 
     steps:
-    - uses: actions/checkout@v2.2.0
-    - name: install ninja
-      run: |
-        mkdir -p ${GITHUB_WORKSPACE}/ninja-bin; cd ${GITHUB_WORKSPACE}/ninja-bin
-        wget https://github.com/ninja-build/ninja/releases/download/v1.9.0/ninja-linux.zip
-        unzip ninja-linux.zip
-        rm ninja-linux.zip
-        echo "${GITHUB_WORKSPACE}/ninja-bin" >> "$GITHUB_PATH"
+    - uses: actions/checkout@v2
+
+    - name: Setup Ninja
+      uses: ashutoshvarma/setup-ninja@v1.1
```

Comment:
As they say, a little bit of copy-paste is better than a little bit of dependency... what does this do that the command you delete doesn't?

Reply:
Well, somebody else maintains it and it looks much cleaner to express the intent. I can go back to the previous version if you insist.

Reply:
Please do if you don't mind. I'd prefer avoiding this dependency (cue the `left-pad` NPM package)